docs/source/app-configuration.rst

```diff
@@ -0,0 +1,157 @@
+App Configuration
+=================
+
+Overview
+--------
+
+For StreamAlert and other related platforms, log forwarding is usually left as an exercise to the reader. This work is non-trivial
+and often requires new infrastructure and code. We wanted to make this easier for everyone and have achieved this through StreamAlert Apps.
+
+StreamAlert Apps allow you to collect logs from popular services and applications in minutes. You simply provide the application's
+credentials and StreamAlert will deploy an individual serverless application that will fetch and forward logs to StreamAlert for analysis and alerting.
+
+
+Concepts
+--------
+
+StreamAlert Apps are made possible through the use of AWS technologies:
+
+* `AWS EC2 Simple System Manager's Parameter Store <https://aws.amazon.com/ec2/systems-manager/parameter-store/>`_
+* `AWS Lambda Invocations via Scheduled Events <http://docs.aws.amazon.com/lambda/latest/dg/with-scheduled-events.html>`_
+
+
+Supported Services
+------------------
+
+* Duo
+
+  - Authentication Logs
+  - Administrator Logs
+
+* *Soon to come:* `OneLogin <https://github.com/airbnb/streamalert/issues/347>`_, `GSuite <https://github.com/airbnb/streamalert/issues/348>`_, and more
+
+
+Getting Started
+---------------
+
+An initial deploy of StreamAlert must be performed before StreamAlert Apps can be configured. If you do not have a current deploy,
```

Comment:
`If you do not have a current deploy` -> `If you haven't deployed StreamAlert yet ...`